kernelci/build.py

```diff
@@ -784,13 +785,13 @@ def install_kernel(kdir, tree_name, tree_url, git_branch, git_commit=None,
         json.dump({'dtbs': sorted(dtb_list)}, json_file, indent=4)
 
     modules_tarball = None
-    if mod_path:
-        abs_mod_path = os.path.join(output_path, mod_path)
-        if os.path.exists(abs_mod_path):
-            modules_tarball = 'modules.tar.xz'
-            modules_tarball_path = os.path.join(install_path, modules_tarball)
-            shell_cmd("tar -C{path} -cJf {tarball} .".format(
-                path=abs_mod_path, tarball=modules_tarball_path))
+    if not mod_path:
```

Comment:
Same comment, default values for the arguments should be set at the top of the function.